docs/guides/update-guide/810-to-820.md

```diff
@@ -0,0 +1,21 @@
+---
+id: 810-to-820
+title: Update 8.1 to 8.2
+description: "Review which adjustments must be made to migrate from Camunda Platform 8.1.x to Camunda Platform 8.2.0."
+---
+
+<span class="badge badge--primary">Intermediate</span>
+
+The following sections explain which adjustments must be made to migrate from Camunda Platform 8.1.x to 8.2.0 for each component.
```

Comment:
Do you anticipate this will outline additional components, or are there any adjustments to include for other components? If not, I would probably just say something like "This guide explains changes and potential adjustments in migrating from Camunda Platform 8.1.x to 8.2.0," remove the H2 "Server", and just have `## Zeebe`.

Reply:
I don't know about other components. But I would assume they would also have some notes for the new version. I have removed the section #Server anyway.